src/harness/testcases/testdata/sas_impl_record_0.json

```diff
@@ -0,0 +1,23 @@
+{
+  "id": "sas_impl/0/0",
```

Comment:
Yup, so you'd use something like "sas_impl/test_admin_name0/unique_impl_id0" as test data, right?